### PR TITLE
[WIP]Fixes#498 Rectify Travis CI tests

### LINKED ISSUE #498: Rectify Travis ci tests
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

Travis ci runs code intended for the old architecture.

**Expected behavior:**

It should support the uncoupled architecture. It should run tests for the front-end and backend on isolated instances.i.e, by making use of the **isolated** parallel builds.

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes #498 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Rectify Travis CI as per new architecture

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,46 +1,112 @@
...
+      - nosetests v1/app/tests/test.py -v --with-coverage
+      - cd v1/cli
+      - python setup.py install
+      - cd ..
+      - badgeyay -i v1/app/static/uploads/yellow_futuristic.png -n sample/businesspass.png.csv
+
+    after_success:
+      - bash <(curl -s https://codecov.io/bash)
+
+  - language: python3
```
Comment (S2606) [inline .travis.yml]: @ParthS007 I think we need only one test for python backend and not two..

File: .travis.yml [hunk 2]
```diff
@@ -1,46 +1,112 @@
...
-python:
-  - 3.5
-cache:
-  directories:
-    - $HOME/.pip-cache/
+    python:
+      - 3.5
+    cache:
+      directories:
+        - $HOME/v1/.pip-cache/
```
Comment (abishekvashok) [inline .travis.yml]: Why should we run on v1,the aim is to test front-end and backend. So pip cache must be present on the backend.

Comment (S2606) [reply .travis.yml]: @abishekvashok Exactly what I mentioned before..

File: .travis.yml [hunk 3]
```diff
@@ -1,46 +1,67 @@
...
+      - sleep 3
+
+    script:
+      - python backend/app/main.py >> log.txt 2>&1  &
+      - nosetests backend/app/tests/test_api.py -v --with-coverage
+
+    after_success:
+      - bash <(curl -s https://codecov.io/bash)
+
+  - language: node_js
```
Comment (S2606) [inline .travis.yml]: @ParthS007 I do think we need two includes for different languages?? Tell me if i am wrong

Comment (abishekvashok) [reply .travis.yml]: @S2606 That's a Travis ci matrix there. A matrix has only 1 include aa i can see.

Comment (S2606) [reply .travis.yml]: Because apparently python backend is not tested.

Comment (S2606) [reply .travis.yml]: @abishekvashok Got it!!

Comment (abishekvashok) [reply .travis.yml]: Cool :)

**Comment by ParthS007** [comment]
making a new and clean PR 👍

**Pull request closed without merging**